D/D.sublime-syntax

```diff
@@ -1191,12 +1198,16 @@ contexts:
         - match: '\('
           scope: punctuation.section.parens.begin.d
           set:
-            - match: '\b{{name}}\b'
-              scope: variable.function.d
+            - match: '\b({{traits}})\b'
+              scope: variable.function.d constant.other.d
```

Comment:
Why stacking `variable.function` and `constant.other`?

Reply:
So that the syntax highlighting falls back in case `constant.other` is not supported. If you prefer just `constant.other` then I can change it.

Reply:
`constant.other` is a very basic scope required to be implemented by each color scheme. There's no fallback required. Actually this combination is just odd.